src/main/java/pl/ateam/disasteralerts/util/EntityAudit.java

```diff
@@ -6,12 +6,14 @@
 import jakarta.persistence.PreUpdate;
 import jakarta.persistence.Version;
 import lombok.Getter;
+import lombok.Setter;
 import org.hibernate.annotations.UuidGenerator;
 
 import java.util.Date;
 import java.util.UUID;
 
 @Getter
+@Setter
```

Comment:
Czemu dodany Setter? Nie powiśmy ingerować i zmieniać np. Id, createDate o czym mówią tesz nazwy metod get****

Reply:
Wynika to z nie wiedzy czemu mapper wcześniej nie działał, teraz oczywiście do usunięcia